### PR TITLE
Follow feature implementation

### PR DESCRIPTION
**Pull request by Nikolaj787**
## Things that have been added!
- A new follow button has been added
- Logic has been added to switch between follow & unfollow button
- A new attribute in Author has been added. Authors now have a list of the authors they have followed. Whenever a user clicks follow, they will now add the author to their own list.
- Usertimelines now displays followed authors cheeps in their own usertimelines. 

## Bugs fixed
- Users are now actually registered as authors whenever they register (Before hand it was only when the user had written a cheep)

### CODE REVIEW

**Review by Tipskind — APPROVED** [review]
I approve this. Some of it is my work, and the other code looks good.

File: src/Chirp.Web/Pages/UserTimeline.cshtml.cs [hunk 1]
```diff
@@ -32,8 +34,22 @@ public async Task<IActionResult> OnGet(string author, [FromQuery] int page)
 
         CurrentAuthor = author;
         PageNumber = page;
-        Cheeps = await _service.GetCheepsFromAuthor(author, page);
+        
+        if (User.Identity != null && User.Identity.Name == author)
```
Comment (niko391a) [inline src/Chirp.Web/Pages/UserTimeline.cshtml.cs:38]: Nice way of ensuring that the logged-in user's messages aren't displayed when visiting others timelines.

**Review by niko391a — APPROVED** [review]
Very nice implementation, and stylish CSS ;)

**Review by DragonFlyersx — APPROVED** [review]
Good implementation and nice to ses the new css

**Pull request merged**